can-construct.js

```diff
@@ -663,6 +709,40 @@ assign(Construct, {
 		 * console.log(Counter.count); // 1
 		 * ```
 		 */
+	},
+	/**
+	 * @function can-construct.ReturnValue ReturnValue
+	 * @parent can-construct.static
+	 * 
+	 * A constructor function which `value` property will be used as a value for a new instance. 
```

Comment:
Too much detail for a description. It shouldn't care at all about the `value` property. Maybe something like:

> Use to overwrite the return value of `new Constructor(...)`.